packages/relay/src/lib/db/types/hbarLimiter/subscriptionTier.ts

```diff
@@ -33,4 +33,9 @@ export enum SubscriptionTier {
    * Trusted Partners
    */
   PRIVILEGED = 'PRIVILEGED',
+
+  /**
+   * Relay Operators
+   */
+  OPERATOR = 'OPERATOR',
```

Comment:
I think this requires an update to the design doc.
Did this already happen or will it be done in follow up?

Reply:
I suggest that we do it in a separate PR to not clutter this one even more with changes

Reply:
Agreed